modules/distribution/rke/main.tf

```diff
@@ -42,6 +42,11 @@ resource "rke_cluster" "this" {
     ssh_key      = var.bastion_host != null ? var.bastion_host.ssh_key : ""
   }
 
+  authentication {
+    sans     = var.additional_hostnames
```

Comment:
When no additional hostnames are present, the `var.additonal_hostnames` variable defaults to `null`. However, this field does not accept a null value.

Reply:
Completed in https://github.com/rancher/tf-rancher-up/pull/147/commits/25311fc91362bb86a1abddd6a9f44be29d0a6786